### PR TITLE
Vaccine branch

### PR DESCRIPTION
**Pull request by troy9393**
Added vaccine branch entities

### CODE REVIEW

**Review by rvmendillo — APPROVED** [review]

**Review by jonnliwagg — APPROVED** [review]
approved

File: src/main/java/com/champets/fureverhome/vaccine/Vaccine.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.champets.fureverhome.vaccine;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
```
Comment (james171001) [inline src/main/java/com/champets/fureverhome/vaccine/Vaccine.java]: Change to @Data

File: src/main/java/com/champets/fureverhome/vaccine/VaccineHistory.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.champets.fureverhome.vaccine;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
```
Comment (james171001) [inline src/main/java/com/champets/fureverhome/vaccine/VaccineHistory.java]: Change to @Data

**Review by james171001 — CHANGES_REQUESTED** [review]

**Review by james171001 — APPROVED** [review]
looks good to me, salary increase next week

**Pull request merged**